app/export.go

```diff
@@ -56,35 +57,71 @@ func (app *NewApp) prepForZeroHeightGenesis(ctx sdk.Context, jailWhiteList []str
 		whiteListMap[addr] = true
 	}
 
+	/* Handle fee distribution state. */
```

Comment:
Can we add some tests around this export data code. Please?

Reply:
Yup, of course. See above. We should add more stuff too.

Reply:
added a shell script to check state is exported properly .